plugin/pkg/scheduler/schedulercache/cache.go

```diff
@@ -439,7 +439,7 @@ func (cache *schedulerCache) cleanupAssumedPods(now time.Time) {
 			panic("Key found in assumed set but not in podStates. Potentially a logical error.")
 		}
 		if !ps.bindingFinished {
-			glog.Warningf("Couldn't expire cache for pod %v/%v. Binding is still in progress.",
+			glog.V(4).Infof("Couldn't expire cache for pod %v/%v. Binding is still in progress.",
```

Comment:
This can be useful if binding takes a long time, but I understand that it could be too verbose in the cases that extenders take longer to bind pods. Please change verbosity to V(1).

Reply:
Done. Changed to `V(3)` since the scheduler's default verbosity is 2.